eclair-core/src/main/resources/reference.conf

```diff
@@ -86,6 +86,8 @@ eclair {
   fulfill-safety-before-timeout-blocks = 24
   min-final-expiry-delta-blocks = 30 // Bolt 11 invoice's min_final_cltv_expiry; must be strictly greater than fulfill-safety-before-timeout-blocks
   max-block-processing-delay = 30 seconds // we add a random delay before processing blocks, capped at this value, to prevent herd effect
+  outdated-commitment-strategy = "always-request-remote-close" // always-request-remote-close or halt-if-just-restarted
+  unhandled-exception-strategy = "local-force-close" // local-force-close or log-and-stop
```

Comment:
>> maybe the new configuration parameter should just be a boolean stop-node-on-outdated-commitment-after-restart = false or something similar?

> It was a left-over of my previous attempt, I flip-flopped around this and thought keeping an explicit strategy name would be easier to understand ... but it is probably not.

Revisiting this PR, I think explicit names are better because they are self-explaining. Also, there are actually four different strategies, so a boolean would be a bit ambiguous (`true` may mean remote-close or local-close):

```
  stop-on-outdated-commitment = false
  stop-on-unhandled-exception = false
```

Preferred alternative:
```
  outdated-commitment-strategy = "remote-close" // remote-close or stop (NB: the app will only stop if it was recently restarted)
  unhandled-exception-strategy = "local-close" // local-close or stop
```

Reply:
Done with a25bc6a.